scripts/travisci/check_imports.py

```diff
@@ -54,6 +54,8 @@ def get_import_statement(node, lines):
     "mujoco_py.builder",
     "mujoco_py.generated",
     "mujoco_py.generated.const",
+    "OpenGL",
+    "OpenGL.GL",
```

Comment:
This is addressed by #160

Reply:
I was testing this change by rebasing it on #160